pkg/cmd/server/api/types.go

```diff
@@ -980,3 +983,19 @@ type AdmissionConfig struct {
 	// on the master. Order is significant. If empty, a default list of plugins is used.
 	PluginOrderOverride []string
 }
+
+// PodLimitRequestConfig is the configuration for the PodLimitRequest
+// admission controller which overrides user-provided container request/limit values.
+type PodLimitRequestConfig struct {
+	unversioned.TypeMeta
+	// Enabled must be true for the plugin to do anything. It can be
+	// overridden per-project with the annotation openshift.io/PodLimitRequestEnabled
```

Comment:
Annotation is not consistent with our pattern - "quota.openshift.io/pod-request-limit-enabled" is more accurate